tutorials/kubernetes.md

```diff
@@ -1,5 +1,7 @@
 # Running CRI-O on kubernetes cluster
 
+This tutorial assumes you've already installed and setup CRI-O. If you have not, start [here](setup.md#starting-cri-o).
+
 ## Switching runtime from docker to CRI-O
```

Comment:
Let us get rid of this section.

Reply:
I changed it but haven't removed it. PTAL at the new version and let me know what you think @mrunalp